src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/operations/activity_log_alerts.py

```diff
@@ -244,4 +224,4 @@ def _get_alert_settings(client, resource_group_name, activity_log_alert_name, th
             else:
                 return None
         else:
-            raise CLIError(ex.message, ex)
+            raise CLIError(ex.message)
```

Comment:
We probably should cross check for why the CLIError no loner accepts inner exception object

Reply:
CLIError as defined in the old core:
```Python 
class CLIError(Exception):
  pass
```

As defined in Knack:
```Python
class CLIError(Exception):
   def __init__(self, message):
     super(CLIError, self).__init__(message)
```

So the reason is because Knack overrides the __init__ and doesn't accept the same arguments.  @derekbekoe is there a reason Knack does this?

Reply:
I don't see a reason why we can't just inherit from Exception like before.
Created https://github.com/Microsoft/knack/pull/65 to fix this.